src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Sort } from './utils/Sort';
+import { TodosList } from './components/TodosList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
 
-const USER_ID = 0;
+const USER_ID = 10360;
 
 export const App: React.FC = () => {
+  const [newTodo, setNewTodo] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortType, setSortType] = useState<Sort>(Sort.All);
+  const [hasError, setHasError] = useState(false);
+
+  const loadTodos = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setHasError(true);
+      setTimeout(() => {
+        setHasError(false);
+      }, 3000);
```

Comment:
you can move this part into `Error` element and hide any error after 3000ms

Reply:
To be honest, I didn't get the idea.